docs/docs/admin/teamforcapella/full-setup/full-setup.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+This tutorial will guide you through the full setup of a new TeamForCapella
+based project in the Collaboration Manager.
+
+1. Create a new project:
+   [Create a new project](../../../user/projects/create.md)
+1. [Create new tool model](../../../user/projects/models/create.md), select
```

Comment:
Maybe "Create a new tool model in the project you created", and move step 4 into step 3 as a parenthetical. If you're not taking an action, I wouldn't include it as a step, and step 4 is critical in order to succeed at step 3 (not everyone reads ahead 😆)

Reply:
1. Renamed to "Create a new tool model in the project you created"
2. Step 4 is not directly related to step 3, so I'd not move them together. However, I clarified why step 3 is important and moved it down in the list (it's not required at that point, but later on).